src/libraries/System.Net.Security/src/System/Net/Security/SslStream.Implementation.cs

```diff
@@ -885,7 +607,7 @@ private void FinishHandshakeRead(int newState)
                 }
 
                 _lockReadState = LockRead;
-                HandleQueuedCallback(ref _queuedReadStateRequest);
+                //HandleQueuedCallback(ref _queuedReadStateRequest);
```

Comment:
Either the code should be restored, or deleted, but not commented out (at least, not without a comment of why code just like this is obvious, but wrong, here).

Reply:
missed & removed.